### ISSUE TITLE
Fresh install of beta, some bots dead...

### ISSUE THREAD

**Issue by Th4nat0s** [certtools/intelmq#217]
After a fresh reinstall of the beta, some parser don't work for me anymore...

```
# python -m intelmq.bots.experts.deduplicator.expert deduplicator-expert
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/lib/python2.7/dist-packages/intelmq-1.0.0-py2.7.egg/intelmq/bots/experts/deduplicator/expert.py", line 5, in <module>
    from intelmq.bots import utils
ImportError: cannot import name utils
```

And, indeed the utils is in lib
/dist-packages/intelmq-1.0.0-py2.7.egg/intelmq/lib

so it should be 
  from intelmq.lib import utils

Isn't it ??  something was mixed with master before.

**Comment by SYNchroACK** [comment]
I still need to test the experts and outputs. But yes, you re right about the solution.
Fixed 36ef20aada1a6d82a3de2eb223659e6035b59bfb

**Comment by Th4nat0s** [comment]
I got the issue in beta 1 ! :)
